onnxmltools/convert/sklearn/operator_converters/NaiveBayes.py

```diff
@@ -0,0 +1,210 @@
+    prob_tensor_name = scope.get_unique_variable_name('prob_tensor')
+
+    class_type = onnx_proto.TensorProto.STRING
+    zipmap_attrs = {'name': scope.get_unique_operator_name('ZipMap')}
+    if np.issubdtype(nb.classes_.dtype, float):
+        class_type = onnx_proto.TensorProto.INT32
+        classes = np.array(list(map(lambda x: int(x), classes)))
+        zipmap_attrs['classlabels_int64s'] = classes 
+    elif np.issubdtype(nb.classes_.dtype, int):
+        class_type = onnx_proto.TensorProto.INT32
+        zipmap_attrs['classlabels_int64s'] = classes
+    else:
+        zipmap_attrs['classlabels_strings'] = classes
+        classes = np.array([s.encode('utf-8') for s in classes])
+
+    container.add_initializer(output_shape_name, onnx_proto.TensorProto.INT64,
+                              [len(output_shape)], output_shape)
+    container.add_initializer(class_log_prior_name, onnx_proto.TensorProto.FLOAT,
+                              class_log_prior.shape, class_log_prior.flatten())
+    container.add_initializer(feature_log_prob_name, onnx_proto.TensorProto.FLOAT,
+                              feature_log_prob.shape, feature_log_prob.flatten())
+    container.add_initializer(classes_name, class_type, 
+                              classes.shape, classes)
+
+    if operator.type == 'SklearnMultinomialNB':
+        matmul_result_name = scope.get_unique_variable_name('matmul_result')
+
+        container.add_node('MatMul', [operator.inputs[0].full_name, feature_log_prob_name],
+                           matmul_result_name, name=scope.get_unique_operator_name('MatMul'))
+        # Cast is required here as Sum op doesn't work with Float64
+        container.add_node('Cast', matmul_result_name, 
+                            cast_result_name, to=onnx_proto.TensorProto.FLOAT, op_version=7)
+        container.add_node('Sum', [cast_result_name, class_log_prior_name],
+                           sum_result_name, name=scope.get_unique_operator_name('Sum'))
+    else:
+        constant_name = scope.get_unique_variable_name('constant')
+        exp_result_name = scope.get_unique_variable_name('exp_result')
+        sub_result_name = scope.get_unique_variable_name('sub_result')
+        neg_prob_name = scope.get_unique_variable_name('neg_prob')
+        sum_neg_prob_name = scope.get_unique_variable_name('sum_neg_prob')
+        inp_neg_prob_prod_name = scope.get_unique_variable_name('inp_neg_prob_prod')
+        difference_matrix_name = scope.get_unique_variable_name('difference_matrix')
+
+        container.add_initializer(constant_name, onnx_proto.TensorProto.FLOAT,
+                                  [], [1.0])
+
+        container.add_node('Exp', feature_log_prob_name,
+                           exp_result_name, name=scope.get_unique_operator_name('Exp'), op_version=6)
+        container.add_node('Sub', [constant_name, exp_result_name],
+                           sub_result_name, name=scope.get_unique_operator_name('Sub1'))
```

Comment:
To support multiple-onnx versions, please use this function: [https://github.com/onnx/onnxmltools/blob/5aabc64d6ebe88e96a8f4f5949db404b10ae2218/onnxmltools/convert/common/_apply_operation.py#L261](https://github.com/onnx/onnxmltools/blob/5aabc64d6ebe88e96a8f4f5949db404b10ae2218/onnxmltools/convert/common/_apply_operation.py#L261)